noise/README.md

```diff
@@ -1,14 +1,16 @@
 # noise-libp2p - Secure Channel Handshake
 
-> A libp2p transport secure channel handshake built with the Noise Protocol Framework.
+> A libp2p transport secure channel handshake built with the Noise Protocol
+> Framework.
 
 | Lifecycle Stage | Maturity      | Status | Latest Revision |
 |-----------------|---------------|--------|-----------------|
-| 1A              | Working Draft | Active | r0, 2019-08-05  |
+| 1A              | Working Draft | Active | r1, 2019-01-20  |
```

Comment:
Changing the year to 2020?

Reply:
haha, good catch 👍

Reply:
Haha, oops, I leaked the time travel transport we were working on secretly.

Fixed: https://github.com/libp2p/specs/pull/244/commits/0f197d2ceec19fb15560f8d29921c35a0f56751e

Reply:
I knew it! Both Noise and libp2p are 2030's tech!